internal/cli/atlas/quickstart/quick_start.go

```diff
@@ -244,6 +244,15 @@ func (opts *Opts) Run() error {
 		return dErr
 	}
 
+	r1, r2, err := opts.store.ProjectClusters(opts.ConfigProjectID(), &atlas.ListOptions{
```

Comment:
We do not use `setup` for non atlas case for this command.
I guess we can even create new interface that returns only Atlas API?

Reply:
https://github.com/mongodb/mongodb-atlas-cli/blob/04845ef93e04e42bc1327e4d5a1253b6a58cab96/internal/cli/opsmanager/clusters/list.go#L75